src/ba-sak/typer.ts

```diff
@@ -12,6 +12,16 @@ export interface IBegrunnelsedata {
   antallBarnOppfyllerTriggereOgHarNullutbetaling: number;
   maanedOgAarBegrunnelsenGjelderFor: string;
   maalform: Maalform;
+  soknadstidspunkt: string;
+  avtaletidspunktDeltBosted: string;
+  belop: string;
+  sokerFaarUtbetaltUtvidetIPerioden: UtvidetPåSøker;
```

Comment:
Trenger hjelp til å finne et passende navn

Reply:
Hmm, ja nå er det lett å tenke at dette er en boolsk verdi. 

Hva med `sokersRettTilUtvidet` eller bare `utvidetPåSoker`?

Reply:
Jeg liker `sokersRettTilUtvidet`!

Reply:
Da går jeg for den 🦈